src/styles/index.css

```
@@ -0,0 +1,12 @@
+@import url(https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500;700&display=swap);
+
+body,
+h2 {
+  margin: 0;
+  padding: 0;
```

body & h2 doesn't have paddings
```suggestion
body,
h2 {
  margin: 0;
```